wfsim/load_resource.py

```diff
@@ -268,3 +263,29 @@ def reduce_last_dim(self):
         shape[-1] = 1
 
         return DummyMap(const, shape)
+
+
+def clear_cached_configs():
```

Comment:
@zhut19 Is this function used?

Reply:
Ah, this one is somehow in my local master branch, keep mixing into my or. It's not used anywhere.

Reply:
Ok, will you delete it or want to keep it there for an upcoming cache cleanup PR?

Reply:
Actually, I don't see if this will be used officially in the future. Let me delete it.